index.html

```diff
@@ -11,32 +11,201 @@
 			body {
 				font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Helvetica Neue", Arial, sans-serif;
 			}
+
+			#search-field-label {
+				display: block;
+			}
+
+			.container {
+				max-width: 1028px;
+				margin: 0 auto;
+			}
+
+			.page-title, .search-form {
+				text-align: center;
+			}
+
+			.article-image {
+				height: 127px;
+				width: 190px;
+				flex: 1;
+			}
+
+			.text-container {
+				flex: 3;
+				margin-left: 1em;
+			}
+
+			.result-item {
+				display: flex;
+				margin: 2em;
+				padding: 1em;
+				border: 2px solid rgba(0, 0, 0, .5);
+			}
+
+			#search-field, #search-button {
+				height: 2em;
+				font-size: 18px;
+			}
+
+			#search-button {
+				margin-left: 1em;
+				display: inline-block;
+			}
+
 		</style>
 	</head>
 
 	<body>
-		<h1>Thrilling Frontend Technical Challenge</h1>
+		<div class="container">
+			<h1 class="page-title">Thrilling Frontend Technical Challenge</h1>
+
+			<form class="search-form">
+				<input type="text" placeholder="Enter Search Term" id="search-field" />
+				<input type="button" value="Search" id="search-button" />
+			</form>
 
-		<h2>Open your Developer Tools Console to view output.</h2>
+			<div id="results-list"></div>
+		</div>
 
 		<script>
-			var request = new XMLHttpRequest();
 
-			request.open('GET', 'https://api.nytimes.com/svc/topstories/v2/science.json?api-key=Gwxln5M3geWlhR6UE0TY1FUWKSG3wCil');
+			const resultsList = document.querySelector('#results-list');
+			const searchField = document.querySelector('#search-field');
+			const searchButton = document.querySelector('#search-button');
+
+			searchField.addEventListener('keydown', (event) => {
+				if (event.keyCode === 13) {
+					event.preventDefault();
+					const request = setupRequest();
+					request.send();
+				}
+			});
+
+			searchButton.addEventListener('click', (event) => {
+				event.preventDefault();
+
+				const request = setupRequest();
+				request.send();
+			});
+
+
+			function setupRequest () {
```

Comment:
How often do we think the results will be updated server side? If we don't care about having down-to-the-second realtime results we could consider running setup on page load, then manipulating the returned data upon form submission. That should increase the perception of responsiveness for the user. We could also refresh the data from the server on an interval.

Reply:
I was thinking about this too. The results actually updated in the time I was taking the test which is what led me to not worry about "caching" the data so to speak. I agree it would make a better experience to have one set of data though so people can actually explore it without weird results! 

Definitely would be cool to update it on an interval and store in local storage or something, or just as a variable. That decision would be up to use case in the end though I guess!